LayoutTests/interaction-region/input-type-range-region-expected.txt

```diff
@@ -12,6 +12,8 @@
         (rect (0,0) width=800 height=600)
 
       (interaction regions [
+        (occlusion (48,-8) width=37 height=36)
+        (borderRadius 0.00),
```

Comment:
we are covering everything with these existing cases, but I'm afraid we could loose this coverage during an unrelated html forms change.

can we add a small guard-focused test?

it would also be a good place to test that we're **not** generating guards for small occlusion layers